kubernetes/helm_charts/logging/fluent-bit/templates/serviceaccount.yaml

```diff
@@ -4,7 +4,11 @@ metadata:
   name: fluent-bit
   namespace: {{ default .Values.namespace .Release.Namespace  }}
 ---
+{{- if  (.Capabilities.APIVersions.Has "rbac.authorization.k8s.io/v1") }}
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
@ddevadat are these conditions specific to OKE?

Reply:
This is added as extra step...so that helm will check whats the current api version of rbac.authorization.k8s.io
( whether v1beta1 or v1 ) and use it accordingly.
Its not specific to OKE